tests/snippets/ints.py

```diff
@@ -97,14 +97,40 @@
 assert -10 // -4 == 2
 
 assert int() == 0
-assert int("101", 2) == 5
-assert int("101", base=2) == 5
```

Comment:
Its good to keep this check with an explicit base given.

Reply:
ok, I'll fix it in a moment